src/js/view/controls/rightclick.js

```diff
@@ -50,6 +51,12 @@ export default class RightClick {
                 link: 'http://www.adobe.com/software/flash/about/'
             });
         }
+        if (!OS.mobile) {
```

Comment:
we should check `if (this.shortcutsTooltip)` instead. That way, we keep all the conditions required for `this.shortcutsTooltip` to be instantiated in one place. Say we decide to exclude it from a specific browser, you would have to add that condition in multiple places.

Reply:
Agreed, I think it's a more logical way to do it. Fixed!